apollos-church-api/src/data/ContentItem.js

```diff
@@ -8,6 +8,14 @@ const schema = gql`
     isFeatured: Boolean
     subtitle: String
     staff: Group
+    ctaLinks: [CTA]
+  }
+
+  type CTA {
```

Comment:
@redreceipt Do we also need an image source?

Reply:
oh yea good call